src/components/AddressInputPanel/index.tsx

```diff
@@ -72,25 +72,31 @@ const DropdownIcon = styled(DropdownSVG)<{ open: boolean }>`
   transform: rotate(${({ open }) => (open ? '-180deg' : 0)});
 `
 
-export const AddressInput = ({
-  onChange,
-  value,
-  error = false,
-  placeholder,
-  icon,
-  disabled = false,
-  style = {},
-  className,
-}: {
+type Props = {
   onChange?: (e: ChangeEvent<HTMLInputElement>) => void
+  onFocus?: (e: ChangeEvent<HTMLInputElement>) => void
+  onBlur?: (e: ChangeEvent<HTMLInputElement>) => void
```

Comment:
```suggestion
  & Pick<DOMAttributes< HTMLInputElement>, 'onBlur' | 'onFocus' | 'onChange'>
```

Reply:
meet this issues when follow your suggestion
<img width="646" alt="image" src="https://user-images.githubusercontent.com/33005392/217416998-669541bd-082b-4834-9086-3acb15bc42b3.png">

Reply:
nice